README.md

```diff
@@ -71,6 +71,7 @@ correctly.
 | [Vundle][vundle]       | `Bundle 'wellle/targets.vim'`                                                 |
 | [Vim-plug][vim-plug]   | `Plug 'wellle/targets.vim'`                                                   |
 | [Pathogen][pathogen]   | `git clone git://github.com/wellle/targets.vim.git ~/.vim/bundle/targets.vim` |
+| [Dein](https://github.com/Shougo/dein.vim) | 		`call dein#add('wellle/targets.vim')`		 |
```

Comment:
Could you use the `[Dein][dein]` syntax here like we do for the others (like `pathogen`) above? So you'd also need to add the `dein` link below, similarly to how the `pathogen` link is set up.